### ISSUE TITLE
Vehicle Table Cruncher -working_dir parameter not working

### ISSUE THREAD

**Issue by wobatt** [volition-inc/Kinzies-Toy-Box#1]
When writing the output cvtf_pc file, the cruncher doesn't include a backslash between the working directory path and the file name.

Example: 
`vehicle_table_crunch.exe -working_dir "foo" -vehicle "foo\car_4dr_compact01_veh.xtbl" -cust_table "foo\car_4dr_compact01_cust.xtbl" -variant_table "foo\car_4dr_compact01.xtbl"`

Writes to:
`foocar_4dr_compact01_veh.cvtf_pc`

Rather than:
`foo\car_4dr_compact01_veh.cvtf_pc`

**Comment by TulipSniper** [comment]
Wild guess, but the tool may be concactinating the strings directly internally. You might be able to work around it by switching to forward slash ("/") for your directories or using a double backslash ("\\\\") in your arguments.

**Comment by wobatt** [comment]
I agree it must be internally concatenating. Adding a trailing backslash to the working directory parameter just gets an error. I've tried both of your suggestions and both get the same result as before.

I have found a work-around by moving to the desired output folder (`cd foo`), then using empty quotes (`-working_dir ""`).
